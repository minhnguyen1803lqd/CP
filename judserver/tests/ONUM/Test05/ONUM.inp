967513927 365180541 540383427
