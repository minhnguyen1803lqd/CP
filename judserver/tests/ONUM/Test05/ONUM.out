169511231905187551
